### PR TITLE
Fix Issue #854 - Range slider, min value over max value.

### PR DESCRIPTION
**Pull request by jespirit**
Pull Requests
=============
Please accompany all pull requests with the following (where appropriate):

- [x] unit tests (we use [Jasmine 2.x.x](https://jasmine.github.io/2.2/introduction))
- [x] JSFiddle (or an equivalent such as CodePen, Plunker, etc) or screenshot/GIF with new feature or bug-fix
- [x] Link to original Github issue (if this is a bug-fix)
- [ ] documentation updates to README file
- [ ] examples within [/tpl/index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl) (for new options being added)
- [ ] Passes CI-server checks (runs automated tests, JS source-code linting, etc..). To run these on your local machine, type `grunt test` in your Terminal within the bootstrap-slider repository directory

This pull request fixes https://github.com/seiyria/bootstrap-slider/issues/854. You can check the issue for my JSFiddle that is used to reproduce the bug.

I added 2 unit tests as well as making a minor edit with a unit test in the same file: `test/specs/DraggingHandlesSpec.js`.

I will say that I was getting this error `Fatal error: Callback must be a function` displayed at the very end in red text, after running the unit tests using `grunt test` command. I don't know why, thoughts?

### CODE REVIEW

File: src/js/bootstrap-slider.js [hunk 1]
```diff
@@ -1659,6 +1659,9 @@ const windowIsDefined = (typeof window === "object");
 				this._setDataVal(val);
 				this._trigger('slideStop', val);
 
+				// No longer need 'dragged' after mouse up
+				delete this._state.dragged;
```
Comment (rovolution) [inline src/js/bootstrap-slider.js]: generally im not a big fan of removing properties with the `delete` keyword (it has dangling memory reference issues in multiple browsers).

Can you refactor this to just set dragged to `null` or something like that instead?

**Comment by rovolution** [comment]
@seiyria personally, im not too picky about fixing the code climate issue, and have debated just removing it altogether since i feel it doesnt deliver any real value TBH.

**Comment by seiyria** [comment]
@rovolution That's fine with me. I'd rather replace it with a linter if we had that set up or some-such. Or just not have it altogether.

**Comment by rovolution** [comment]
we already have a linter setup in our CI check - https://github.com/seiyria/bootstrap-slider/blob/master/Gruntfile.js#L274-L278

File: src/js/bootstrap-slider.js (hunk 1 above)
Comment (jespirit) [reply src/js/bootstrap-slider.js]: I was just copying the behaviour of the keydown event, that will set and delete `this._state.keyCtrl`. I will set it to null, it doesn't seem to break any of the logic.

https://github.com/seiyria/bootstrap-slider/blob/2272d5a0249b6ccb05efc2d5eced52720d8272aa/src/js/bootstrap-slider.js#L1565

**Comment by rovolution** [comment]
@jespirit LGTM 👍 ill merge, publish, and update changelog.

**Comment by rovolution** [comment]
@jespirit merged and published to v10.2.3.

Thanks again for your contribution!

**Pull request merged**